src/frontend/src/components/ui/sidebar.tsx

```diff
@@ -0,0 +1,772 @@
+"use client";
```

Comment:
can you remove the "use-client"? since we are not on next there is no real usage to it